Add Github OAuth samples configuration in README.md

akrifari opened the pull request:
Fix #210

### Description
Add Github OAuth samples configuration for new developer who want to contribute

akrifari committed:
Add Github OAuth samples configuration in README.md

Fix #210

byliuyang commented on README.md:
```diff
@@ -23,7 +23,19 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
+| Application Name           | `short`                                                  |
+| Homepage URL               | `https://s.time4hacks.com`                               |
+| Application description    | `URL shortening service written in Go and React`         |
+| Authorization callback URL | `https://s.time4hacks.com/oauth/github/sign-in/callback` |
```
I wonder whether we want to change `https://s.time4hacks.com` to `http://localhost` in order to easy local testing.

bigfpanda replied:
Yeah, agreed

byliuyang commented on README.md:
```diff
@@ -23,7 +23,19 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
+| Authorization callback URL | `https://s.time4hacks.com/oauth/github/sign-in/callback` |
+
+2. Once you have registered the application, you should get `Client ID` and `Client Secret`
+3. Now you can copy the `Client ID` and `Client Secret` into their respective places in the `.env` file
```
```suggestion
3. Now you can copy the `Client ID` and `Client Secret` into their respective locations in the `.env` file
```

byliuyang commented on README.md:
```diff
@@ -23,7 +23,19 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
+
+| Field                      | Value                                                    |
+| -------------------------- | -------------------------------------------------------- |
+| Application Name           | `short`                                                  |
```
```suggestion
| Application Name           | `Short`                                                  |
```

byliuyang commented on README.md:
```diff
@@ -23,7 +23,19 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
-[Create Github OAuth App](https://github.com/settings/developers)
+### Create Github OAuth Application
+
+1. Register in [Github OAuth](https://github.com/settings/developers) with this following examples:
```
```suggestion
1. Register in [Github OAuth](https://github.com/settings/developers) with the following configurations:
```

akrifari committed:
Rename base URL with localhost

akrifari committed:
Replace a few words

akrifari committed:
Change https to http

coderworld10 committed:
Merge branch 'master' into docs/github-oauth-sample

byliuyang commented on README.md:
```diff
@@ -23,7 +23,19 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
-[Create Github OAuth App](https://github.com/settings/developers)
+### Create Github OAuth Application
+
+1. Register in [Github OAuth](https://github.com/settings/developers) with the following configurations:
```
```suggestion
1. Register at [Github OAuth](https://github.com/settings/developers) with the following configurations:
```

byliuyang commented:
Hi @bigfpanda , thank you for working on this issue. Please address Code Climate's comments.

akrifari committed:
Follow code review guidance

- Reformat lines that are exceeded line length limit
- Change numbered list with unordered list
- Replace the word `in` with `at` at first step

coderworld10 committed:
Merge branch 'master' into docs/github-oauth-sample

bigfpanda commented:
Thanks :+1:
